pyoozie/tags.py

```diff
@@ -491,3 +506,55 @@ def _xml(self, doc, tag, text):
                         self.workflow_configuration._xml(doc, tag, text)
 
         return doc
+
+
+class WorkflowApp(XMLSerializable):
+
+    def __init__(
+            self,
+            name,                         # type: typing.Text
+            parameters=None,              # type: typing.Optional[PropertyValuesType]
+            configuration=None,           # type: typing.Optional[PropertyValuesType]
+            credentials=None,             # type: typing.Optional[typing.Iterable[Credential]]
+            job_tracker=None,             # type: typing.Optional[typing.Text]
+            name_node=None,               # type: typing.Optional[typing.Text]
+            job_xml_files=None,           # type: typing.Optional[JobXmlFilesType]
+            default_retry_max=None,       # type: typing.Optional[int]
+            default_retry_interval=None,  # type: typing.Optional[int]
+            default_retry_policy=None     # type: typing.Optional[RetryPolicy]
+    ):
+        # type: (...) -> None
+        XMLSerializable.__init__(self, 'workflow-app')
+        self.name = validate_xml_name(name)
+
+        self.parameters = Parameters(parameters)
+        self.global_configuration = GlobalConfiguration(
+            job_tracker=job_tracker,
+            name_node=name_node,
+            job_xml_files=job_xml_files,
+            configuration=configuration
+        )
+        self.credentials = credentials
+
+        self.default_retry_max = default_retry_max
+        self.default_retry_interval = default_retry_interval
+        self.default_retry_policy = default_retry_policy
```

Comment:
Ah, we shouldn't even include them here probably; this is copy-pasta from a previous incarnation of this API.